wiki/Tournaments/ATT/2021/en.md

```diff
@@ -0,0 +1,143 @@
+---
+tags:
+  - ATT
+  - ATT2021
+---
+
+# Ausu!Taiko Tournament 2021
+
+![ATT 2021 logo](img/logo.jpg)
+
+The **Ausu!Taiko Tournament 2021** (***ATT 2021***) is a double-elimination 1v1 osu!taiko tournament hosted by ![][flag_NZ] [Sparxe](https://osu.ppy.sh/users/5750235). The tournament was restricted to players within the Southeast Asian countries only. It is the seventh installment of the Ausu!Taiko Tournament.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2021-03-29/2021-04-05 |
+| Seedings and groups released | 2021-04-18 |
+| Group Stage | 2021-04-24/2021-04-25 |
+| Round of 16 | 2021-05-01/2021-05-02 |
+| Quarterfinals | 2021-05-08/2021-05-09 |
+| Semifinals | 2021-05-15/2021-05-16 |
+| Finals | 2021-05-22/2021-05-23 |
+
+## Prizes
+
+| Placing | Prize |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge |
+
+## Organisation
+
+The Ausu!Taiko Tournament 2021 is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Organiser | ![][flag_NZ] [Sparxe](https://osu.ppy.sh/users/5750235) |
+| Mappool selector | ![][flag_JP] [Maimaing](https://osu.ppy.sh/users/14520910), ![][flag_NZ] [Sparxe](https://osu.ppy.sh/users/5750235) |
+| Referee | ![][flag_MY] [cdhsausageboy](https://osu.ppy.sh/users/2403621), ![][flag_AU] [Fairy Bread](https://osu.ppy.sh/users/8306102), ![][flag_FR] [Kasumi-sama](https://osu.ppy.sh/users/6177263), ![][flag_SG] [Spartric](https://osu.ppy.sh/users/7740442), ![][flag_MY] [Xeltic Rival](https://osu.ppy.sh/users/7500364) |
+| Streamer | ![][flag_FR] [Kasumi-sama](https://osu.ppy.sh/users/6177263), ![][flag_SG] [Spartric](https://osu.ppy.sh/users/7740442) |
+| Designer | ![][flag_PH] [OsuMe65](https://osu.ppy.sh/users/852867) |
+| Wiki editor | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1284274)
+- [Livestream](https://www.twitch.tv/sparxenic)
+- [Challonge bracket](https://challonge.com/ausutaiko2021)
+- Spreadsheet
+  - [Schedule](https://docs.google.com/spreadsheets/d/18xadwgbLM3mc5nPs316bF0OSwcyyGSgibFEv_hNkgeU/edit?usp=sharing)
+  - [Mappool](https://docs.google.com/spreadsheets/d/1dUo00W0UH3pgq1wPru-U9EGf35qAisRyM2C6OWBn2Ns/edit?usp=sharing)
```

Comment:
```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1284274)
- [Livestream](https://www.twitch.tv/sparxenic "Twitch")
- [Challonge bracket](https://challonge.com/ausutaiko2021 "Challonge")
- Spreadsheet
  - [Schedule](https://docs.google.com/spreadsheets/d/18xadwgbLM3mc5nPs316bF0OSwcyyGSgibFEv_hNkgeU/edit?usp=sharing "Google Docs")
  - [Mappool](https://docs.google.com/spreadsheets/d/1dUo00W0UH3pgq1wPru-U9EGf35qAisRyM2C6OWBn2Ns/edit?usp=sharing "Google Docs")
```

Reply:
Will also do this in later tourney articles, just knew.